<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConnect</class>
 <widget class="QDialog" name="DialogConnect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Соединение...</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/about/icons/logo.png</normaloff>:/about/icons/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel4">
     <property name="text">
      <string>Пользователь</string>
     </property>
     <property name="buddy">
      <cstring>editUsername</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="editUsername">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="textLabelHostname">
     <property name="text">
      <string>Сервер</string>
     </property>
     <property name="buddy">
      <cstring>editHostname</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLineEdit" name="editHostname">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="textLabelPort">
     <property name="text">
      <string>Порт</string>
     </property>
     <property name="buddy">
      <cstring>portSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QSpinBox" name="portSpinBox">
     <property name="specialValueText">
      <string>Default</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="textLabelBD">
     <property name="text">
      <string>База данных</string>
     </property>
     <property name="buddy">
      <cstring>editDatabase</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QLineEdit" name="editDatabase">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="textLabelDrv">
     <property name="text">
      <string>Драйвер</string>
     </property>
     <property name="buddy">
      <cstring>comboDriver</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QComboBox" name="comboDriver"/>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="editPassword">
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel4_2">
     <property name="text">
      <string>Пароль</string>
     </property>
     <property name="buddy">
      <cstring>editPassword</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="labelText">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;Введите&lt;b&gt; имя пользователя&lt;/b&gt; и &lt;b&gt; пароль&lt;/b&gt;&lt;br&gt;для инициализации рабочего места&lt;br&gt;системы&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelImage">
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonProp">
       <property name="text">
        <string>Подробно</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>DialogConnect</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogConnect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
